test/example/qt.erl

```diff
@@ -70,14 +70,14 @@ listener(L) ->
 
 acceptor(Top, L) ->
   io:format("Call accept ~p\n", [self()]),
-  {ok, Conn} = quicer:accept(L, [], infinity),
+  {ok, Conn} = quicer:accept(L, #{}, infinity),
 
   Top ! {self(), more},
 
   io:format("accept -> ~p\n",[Conn]),
 
   case quicer:handshake(Conn) of
-    {ok, Conn} ->
+    {ok, _Conn} ->
```

Comment:
why this change?

Reply:
Sigh, stupid, just  a misdirected attempt at getting through dialyzer, skip that